Update muon interface docs (#25781)
mantidproject/mantid: docs/source/interfaces/muon_home_tab.rst

**Time Zero** This is the time zero value used. By default the time zero is taken from the file but
it can be specified here in which case the specified time zero is used for all calculations.

> **First Good Data** This is the first good data value used. By default the first good data is taken from the file but

Reviewer: ```suggestion
**First Good Data** This is the first good data value used. By default the first good data is taken from the file. However, the user can manually change the value.
```

> it can be specified here in which case the specified first good data is used for all calculations.

Reviewer: ```suggestion
The specified first good data is used for all calculations.
```

> **Last Good Data** This is the Last good data value used. By default the last good data is taken from the file but

Reviewer: ```suggestion
**Last Good Data** This is the Last good data value used. By default the last good data is taken from the file. However, the user can manually set the value.
```

> it can be specified here in which case the specified last good data is used for all calculations.

Reviewer: ```suggestion
The specified first good data is used for all calculations.
--

```
